### PR TITLE
handle "Invalid Date" in R.toString

### PR DESCRIPTION
**Pull request by davidchambers**
See plaid/sanctuary-def#11

/cc @svozza

### CODE REVIEW

File: test/toString.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ describe('toString', function() {
 
   it('returns the string representation of a Date object', function() {
     assert.strictEqual(R.toString(new Date('2001-02-03T04:05:06.000Z')), 'new Date("2001-02-03T04:05:06.000Z")');
+    assert.strictEqual(R.toString(new Date('XXX')), 'new Date(NaN)');
```
Comment (davidchambers) [inline test/toString.js]: `'new Date(NaN)'` seems to me the most appropriate string representation of this value.

Comment (CrossEye) [inline test/toString.js]: Agreed

**Comment by CrossEye** [comment]
LGTM

**Comment by svozza** [comment]
+1

**Pull request merged**